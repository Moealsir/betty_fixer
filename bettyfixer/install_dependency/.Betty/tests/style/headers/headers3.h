#ifndef _PROTECTION_H_
#define TEST 5

struct sample_s
{
	char *str;
	int i;
};

#endif
#ifndef _TEST_H_
#define _TEST_H_
union sample_u
{
	char rgba[4];
	int value;
};

enum sample_e
{
	FIRST = 0,
	SECOND,
	THIRD
};
#endif
/* ----- */

typedef struct sample_s sample_s;
typedef union sample_u sample_u;
typedef enum sample_e sample_e;
