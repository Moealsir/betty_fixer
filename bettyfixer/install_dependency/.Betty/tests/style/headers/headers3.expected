tests/style/headers/headers3.h:2: WARNING: This line is not protected from double inclusion
tests/style/headers/headers3.h:4: WARNING: This line is not protected from double inclusion
tests/style/headers/headers3.h:5: WARNING: This line is not protected from double inclusion
tests/style/headers/headers3.h:6: WARNING: This line is not protected from double inclusion
tests/style/headers/headers3.h:7: WARNING: This line is not protected from double inclusion
tests/style/headers/headers3.h:8: WARNING: This line is not protected from double inclusion
tests/style/headers/headers3.h:26: WARNING: This line is not protected from double inclusion
tests/style/headers/headers3.h:28: WARNING: This line is not protected from double inclusion
tests/style/headers/headers3.h:29: WARNING: This line is not protected from double inclusion
tests/style/headers/headers3.h:30: WARNING: This line is not protected from double inclusion
total: 0 errors, 10 warnings, 30 lines checked
